utils/addr_parse_test.go

```diff
@@ -1,11 +1,17 @@
 package utils
 
 import (
-	"github.com/stretchr/testify/assert"
 	"testing"
+
+	"github.com/stretchr/testify/assert"
 )
 
 func TestParseServicePath(t *testing.T) {
 	_, _, err := ParseServicePath("Greeter.Hello")
-	assert.NotNil(t,err)
+	assert.NotNil(t, err)
+
+	seriveName, method, err := ParseServicePath("SerivceName/Method")
+	assert.Equal(t, seriveName, "erivceName")
```

Comment:
Spelling mistake：erivceName should be ServiceName

Reply:
but this function will remove the first characters
such as: 
input: 1234/abcd 
output: 234, abcd, nil

Reply:
the valid seriveName should be /SerivceName/Method.

By the way, you can add a judgment to the ParseServicePath method，

by using strings.HasPrefix to judge whether the string starts with /